packages/utils/src/report/get-css-code-snippet.ts

```diff
@@ -0,0 +1,111 @@
+import { ChildNode } from 'postcss';
+
+const getCodeSnippetPrefix = (node: ChildNode): string => {
+    if (node.type === 'rule') {
+        return `${node.selector} {`;
+    }
+
+    if (node.type === 'atrule') {
+        return `@${node.name} ${node.params} {`;
+    }
+
+    return '';
+};
+
+const getCodeSnippetContent = (node: ChildNode, content: string): string => {
+    let numberOfSpaces = 4;
+    const grandpa = node.parent ? node.parent.parent : null;
+
+    if (grandpa && grandpa.type === 'atrule') {
+        numberOfSpaces = 8;
+    }
+
+    const spaces = new Array(numberOfSpaces + 1).join(' ');
+
+    return `${spaces}${content}`;
```

Comment:
This should be a little clearer (also requires deleting `const spaces = ...` above).

```suggestion
    return content.padStart(numberOfSpaces, ' ');
```

Reply:
I didn't like the `new Array` solution, but I couldn't think in a shorter way to do it. Your solution is much better :)

Reply:
If only I had written it correctly the first time 🤦‍♂️ Thanks for debugging this with me!

```ts
return content.padStart(content.length + numberOfSpaces, ' ');
```